verification/curator-service/api/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../../common/tsconfig.json",
   "compilerOptions": {
     "outDir": "dist", /* Redirect output structure to the directory. */
+    "resolveJsonModule": true
```

Comment:
Did you get an error without this? This is in the base tsconfig.

Reply:
ah my bad, I didn't notice it